ui/raidboss/data/05-shb/dungeon/twinning.js

```diff
@@ -302,10 +302,11 @@
         'Mark CXLIV Thermocoil Boilbuster': 'Magitek-Kessel Typ CXLIV',
         'Engage!': 'Start!',
         'Levinball': 'Donnerkugel',
-        'The Cornice': 'Schnittstelle',
-        'Aetherial Observation': 'Ätherobservationsdeck',
-        'Repurposing': 'Umrüstanlage',
         'Cladoselache': 'Cladoselache',
+        'The Cornice will be sealed off': 'Noch 15 Sekunden bis sich der Zugang zu Schnittstelle schließt',
```

Comment:
Noch 15 Sekunden, bis sich die Schnittstelle schließt.
Noch 15 Sekunden, bis sich das Ätherobservationsdeck schließt.
Noch 15 Sekunden, bis sich die Umrüstanlage schließt.

Again these are directly gathered from the logfiles